pandas/tests/scalar/interval/test_interval.py

```diff
@@ -109,79 +109,103 @@ def test_length_errors(self, left, right):
         with tm.assert_raises_regex(TypeError, msg):
             iv.length
 
-    def test_math_add(self, interval):
```

Comment:
I could have parametrized the `interval` fixture over `closed` but it would have blown up a lot of other tests that use this fixture.  Wanted this PR to be easy to review, so I only made the arithmetic operations related changes.

Will clean up these tests as a whole in a follow up.  Found a few other bugs when reviewing this code and will want to document/fix those and take them into account prior to cleaning.